UI/GameSettingsScreen.cpp

```diff
@@ -381,10 +394,7 @@ void GameSettingsScreen::CreateGraphicsSettings(UI::ViewGroup *graphicsSettings)
 
 #if !(PPSSPP_PLATFORM(ANDROID) || defined(USING_QT_UI) || PPSSPP_PLATFORM(UWP) || PPSSPP_PLATFORM(IOS))
 		CheckBox *vSync = graphicsSettings->Add(new CheckBox(&g_Config.bVSync, gr->T("VSync")));
-		vSync->OnClick.Add([=](EventParams &e) {
-			NativeResized();
-			return UI::EVENT_CONTINUE;
-		});
+        OnClickPostDarwinNotification(vSync, "ConfigDidChange", "VSync")
```

Comment:
nit: Spaces stranded in a land of tabs.  Happens a few other times, but I guess these lines will go away anyway when the menu builds itself.

More importantly, I think we still need the vsync `NativeResized()`, at least on other platforms?

-[Unknown]

Reply:
> More importantly, I think we still need the vsync `NativeResized()`, at least on other platforms?

oops, forgot this, thanks for pointing out